src/libraries/System.IO.FileSystem/tests/Base/FileGetSetAttributes.cs

```diff
@@ -99,7 +99,7 @@ public void GettingAndSettingAttributes_AlternateDataStream_Windows(string strea
         {
             string path = CreateItem();
             streamName = path + streamName;
-            File.Create(streamName);
+            File.Create(streamName).Dispose();
```

Comment:
I think an analyzer-codefix for a standalone call to `File.Create(path)` would be very helpful to avoid these mistakes.

Reply:
It should be found if you change CS2000 to "warn"
https://github.com/danmoseley/runtime/blob/4225cb0fff263e9fbc7ad007ab18e774a1b73d1e/eng/CodeAnalysis.test.globalconfig#L409
CA2213 is also relevant.

@stephentoub did you try these -- was there a lot of noise?